x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/rest/RestSqlTestCase.java

```diff
@@ -1055,6 +1055,16 @@ private void executeQueryWithNextPage(String format, String expectedHeader, Stri
         assertEquals(0, getNumberOfSearchContexts(client(), "test"));
     }
 
+    public void testMultiValueQueryText() throws IOException {
+        index("{\"text\":[" + toJson("one") + "," + toJson("two, three") + "," + toJson("\"four\"") + "], \"number\" : [1, [2, 3], 4] }");
```

Comment:
Why do you use `toJson` only for the values but not also for the field names?

Reply:
C&p inertia. Thx, updated it.